datamade/chi-councilmatic
# Import shapes with new management command

## Overview

Following the changes in https://github.com/datamade/django-councilmatic/pull/249, create a `data` directory to store Post shapes locally, and use the new `import_shapes` management command to load those shapes into the database.

## Testing instructions

### Test the Makefile

- Confirm that you can recreate the data file:
    - `cd data`
    - `make clean`
    - `make`

### Test the `import_shapes` command

- Check out the branch from https://github.com/datamade/django-councilmatic/pull/249 to your local `django-councilmatic` instance, and make sure it's installed to the `chi-councilmatic` virtualenv with `pip install -e /path/to/django-councilmatic`
- Run migrations: `python manage.py migrate`
- Import the shapes: `python manage.py import_shapes data/final/shapes/chicago_shapes.json`
- Run the development server: `python manage.py runserver`
- Navigate to http://localhost:8000/council-members/ and confirm that the boundaries load as expected
- Navigate to a council member detail page (e.g. http://localhost:8000/person/la-spata-daniel-bca2038dd957/) and confirm that the boundaries load as expected
